autonomy/analyse/abci/app_spec.py

```diff
@@ -430,19 +429,22 @@ def check_one(informat: str, infile: str, classfqn: str) -> bool:
 
     @classmethod
     def check_all(cls, packages_dir: Path) -> None:
-        """Check all the available definitions."""
+        """
+        Check all the available definitions.
+
+        This function finds all files whose name is 'fsm_specification.yaml' in an AEA skill package, and checks them
+        one by one using the 'check_one' function.
+
+        :param packages_dir: the package directory where to find the spec files.
+        """
 
         did_not_match = []
         fsm_specifications = sorted(
             [
-                *packages_dir.glob(f"**/{DEFAULT_FSM_SPEC_YAML}"),
+                *packages_dir.glob(f"*/skills/*/{DEFAULT_FSM_SPEC_YAML}"),
             ]
         )
-        # TODO: fix this implementation:
-        # - above file names should be constant and explained in helper text
```

Comment:
- "file names should be constant" this is done even before this PR, right?

- "explained in helper text", done in 9ce34dfd2411f45ea201f16aa20e72ab9ce1438a (not in the helper text of the *CLI command* though)